src/App.js

```diff
@@ -73,8 +68,12 @@ const App = () => {
     const onSortColumn = (columnToSort) => {
         let direction;
 
-        // 1. if exists -->> flip it
-        // 2. if dont exists --> destroy everything in it, create this key
+        // 1. if dont exists --> destroy everything in it, create this key
+        // 2. if exists -->> flip it
+
+        if (sortConfig['columnKey'] !== columnToSort) {
+            direction = 'asc';
+        }
```

Comment:
This if is needed if it has never been sorted

Reply:
Why not just declare it when you init the component?